native/dlassq.go

```diff
@@ -0,0 +1,26 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import "math"
+
+// Dlassq updates a sum of squares in scaled form.
+func (impl Implementation) Dlassq(n int, x []float64, incx int, scale float64, sumsq float64) (float64, float64) {
```

Comment:
Is there any sort of convention for these kind of names? scaleAfter or something?